config/default/manager/manager.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       containers:
       - command:
         - /manager
-        image: github.com/kubeflow/kfserving/cmd/manager
+        image: ko://github.com/kubeflow/kfserving/cmd/manager
```

Comment:
Can you explain this change?

Reply:
I got this warning when running `make deploy-dev` , so all the image path needs to be prefixed with `ko://` now.
```
We will start requiring ko:// in a coming release.  Please prefix
the following import path for things to continue working:

   github.com/kubeflow/kfserving/cmd/logger
```